src/components/TopBar/TopBar.vue

```diff
@@ -46,6 +46,23 @@
 			</div>
 		</a>
 
+		<a v-if="showUpcomingEvent"
+			class="upcoming-event__card"
+			:aria-label="t('spreed', 'Next call')"
```

Comment:
Isn't it duplication to have "Next call" as aria label when it is already in the visible content itself? @ShGKme